### DISCUSSION TITLE
Fehlermeldung in Node Red: token not ok --- Token expired

### DISCUSSION THREAD

**Discussion by vonRepel** [DrHouseIT/everHome2MQTT#25] [Fragen & Antworten (Q&A)]
8 Dec 12:24:39 - [info] [function:Check token - startup] token not ok
8 Dec 12:24:39 - [info] [function:Check token - startup] Token expired: currentTime: Sun Dec 08 2024 12:24:39 GMT+0100 (Central European Standard Time); expiresTime: Sun Dec 08 2024 12:24:32 GMT+0100 (Central European Standard Time)

Moin @All,
moin Daniel,

irgendwie bekomme ich meine Konfiguration nicht wieder zum Laufen.
Habt Ihr /Du einen Tipp, woran es liegen könnte?
Die secret-id  habe ich schon öfters erneuert, auch die gesamte Anwendung gelöscht und einen neuen Eintrag  der secret erzeugt.
Habe nun schon mehrfach die Installation nach der README des updates 2.0.8 durchgeführt.
Komme da einfach nicht weiter.
Gruß
Ralph

**Comment by DrHouseIT-Daniel** [comment]
Hi @vonRepel,
hast du nach diesen Maßnahmen die _OAuth 2.0 authentication for REST API_ durchgeführt?
![image](https://github.com/user-attachments/assets/008b596f-c539-47a5-ada6-2e02892aaee9)
